SDWebImage/UIView+WebCacheOperation.m

```diff
@@ -26,7 +26,7 @@ - (SDOperationsDictionary *)sd_operationDictionary {
         if (operations) {
             return operations;
         }
-        operations = [[NSMapTable alloc] initWithKeyOptions:NSMapTableCopyIn valueOptions:NSMapTableWeakMemory capacity:0];
```

Comment:
Acutally copy is useless. Because we always pass a immutable `NSString` but not `NSMutableString`. Use strong-weak is OK.